### ISSUE TITLE
Survey instance definition

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-tools-dev/cetaceans#240]
from Nikki:

Definition of a survey – attempt 1:

'Survey'

The term 'survey' may be applied differently between organisations, depending on how data are collected. In order to reduce the burden of data uploads and associated metadata, the data submitter may combine multiple 'survey events' into a single 'survey'. Therefore, for this purpose, a 'survey' is:

'Data collection within a single calendar year using a single methodology and/or within a pre-defined area or route'.

SurveyID now: year organisation location (previously date org location surveytype)_

**Comment by neil-ices-dk** [comment]
> 'Data collection within a **single calendar year** using a single methodology and/or within a pre-defined area or route'.

will there be a situation when a contiguous survey is conducted seasonally and therefore traversing a calendar year?

**Comment by NikkiTaylorJNCC** [comment]
> > 'Data collection within a **single calendar year** using a single methodology and/or within a pre-defined area or route'.
> 
> will there be a situation when a contiguous survey is conducted seasonally and therefore traversing a calendar year?

There are seasonal surveys but I would expect them to upload annually ...assume this would therefore be counted as different surveys and that they aren't able to add data to an existing survey...

**Comment by neil-ices-dk** [comment]
Conclusion:
**SurveyID** = year:organisation:location
**Description** = _Data collection within a single calendar year using a single methodology and/or within a pre-defined area or route_

------------------------
Whether to allow multiple surveyID's in one file submission? there are pro's and con's for either approach; for the 'big' submitters of multiple surveys, it could be an idea to have an R script that generates the XML output packaged into files. 
@NikkiTaylorJNCC will investigate if there are resources that can support this development
------------------------

**Comment by neil-ices-dk** [comment]
@cmspinto to make changes to database to reflect new key
@pcrjoana to make changes to Excel Template/format
@NikkiTaylorJNCC to make changes to Data standard
For primary and secondary platforms on same survey; add new field on sightings and effort records to identify which one is which

**Comment by neil-ices-dk** [comment]
@NikkiTaylorJNCC and @pcrjoana 
fieldname = PlatformInstance
dropdown vocab; 'Primary', 'Secondary'

**Comment by neil-ices-dk** [comment]
@cmspinto agreed to have export to multiple files using Excel VBA functionality; agreed that this will need some heavy testing by ICES with real data, and by Lucy with real data

**Comment by neil-ices-dk** [comment]
![image](https://user-images.githubusercontent.com/23527005/148530441-f680d172-2989-4a79-a1ea-b0ca27425bac.png)
change text to 'Errors'
